gallery/src/util/jsUtil.ts

```diff
@@ -12,3 +12,32 @@
 export function ensureArray<T>(items: T | T[]): T[] {
   return Array.isArray(items) ? items : [items];
 }
+
+// Quick and dirty removal of any comment appearing at the beginning of the content, as it is assumed to be
+// a license which is not helpful to the example display. Both HTML-syntax and C-syntax comments are removed
+export const removeLicense = (content: string) =>
+  content
+      .replace(/^<!--(.|\n)*?-->\s*\n?/, '')
+      .replace(/^\/\*(.|\n)*?\*\/\s*\n?/, '');
+
+// Copies a string to the clipboard.
+export const copyTextToClipboard = (textToCopy: string) => {
+  // navigator clipboard api needs a secure context (https)
+  if (navigator.clipboard && window.isSecureContext) {
+    // navigator clipboard api method'
+    navigator.clipboard.writeText(textToCopy);
```

Comment:
This returns a promise that could be rejected.  You'll want to tie that in with your error handling

Reply:
> This returns a promise that could be rejected. You'll want to tie that in with your error handling

Updated to use promises: 82608d9d729df00c7f41d74640faba7edf16da20